clevertap-core/src/main/java/com/clevertap/android/sdk/inapp/customtemplates/CustomTemplateContext.kt

```diff
@@ -132,12 +135,12 @@ sealed class CustomTemplateContext private constructor(
      * visible to the user until this method is called. Since the SDK can show only one InApp message at a time, all
      * other messages will be queued until the current one is dismissed.
      */
-    open fun setDismissed(context: Context) {
+    open fun setDismissed() {
         val listener = inAppListenerRef.get()
         if (listener != null) {
-            listener.inAppNotificationDidDismiss(context, notification, null)
+            listener.inAppNotificationDidDismiss(null, notification, null)
```

Comment:
lets remove context from func signature if it is not needed.

Reply:
This method is also used in InAppNotificationActivity and CTInAppBaseFragment with `Activity::getBaseContext()`. The provided context is then used in the next execution of showInApp to create a launch Intent for InAppNotificationActivity. This context is not used in the case of custom templates and so I removed it from the dismiss of CustomTemplateContext. Overall not a great way to handle this, we can improve it in the future.